src/components/error-browser-not-supported/error-browser-not-supported.spec.ts

```diff
@@ -0,0 +1,164 @@
+import { shallow, Wrapper } from '@vue/test-utils';
+
+import { renderComponent } from '../../../tests/helpers/render';
+import { ERROR_TEMPLATE_NAME } from '../component-names';
+import { Link } from '../error-template/error-template';
+import { MErrorBrowserNotSupported } from './error-browser-not-supported';
+
+let wrapper: Wrapper<MErrorBrowserNotSupported>;
+let isMobileDevice: boolean;
+const A_CUSTOM_TITLE: string = 'An error title.';
+const A_HINT: string = 'aHint';
+const A_LINK: Link = new Link('aLabel', 'anUrl');
+
+const getStubs: any = () => {
+    return {
+        [ERROR_TEMPLATE_NAME]: '<div><slot /></div>'
+    };
+};
+
+const getMocks: any = () => {
+    return {
+        $mq: {
+            register: jest.fn()
+        }
+    };
+};
+const initializeWrapperDefaultValues: () => void = (): void => {
+    wrapper = shallow(MErrorBrowserNotSupported, {
+        mocks: getMocks(),
+        stubs: getStubs(),
+        mixins: [{
+            data: function(): any {
+                return {
+                    isMqMinS: isMobileDevice ? false : true
+                };
+            }
+        }]
+    });
+};
+
+const initializeWrapperCustomValues: () => void = (): void => {
+    wrapper = shallow(MErrorBrowserNotSupported, {
+        stubs: getStubs(),
+        mocks: getMocks(),
+        propsData: {
+            title: A_CUSTOM_TITLE,
+            hints: [A_HINT],
+            links: [A_LINK]
+        },
+        mixins: [{
+            data: function(): any {
+                return {
+                    isMqMinS: isMobileDevice ? false : true
+                };
+            }
+        }]
+    });
+};
+
+beforeEach(() => {
+    isMobileDevice = false;
+});
+describe(`Page not found - test`, () => {
```

Comment:
Page not found? Je pense que c'est censé être "Browser not supported".

Reply:
done